### PR TITLE
Update universe.json

### PR DESCRIPTION
**Pull request by wjbmattingly**
added hobbit-spacy to the universe json

## Checklist

- [x] I confirm that I have the right to submit this contribution under the project's MIT license.
- [x] I ran the tests, and all new and existing tests passed.
- [x] My changes don't require a change to the documentation, or if they do, I've added all required information.

### CODE REVIEW

File: website/meta/universe.json [hunk 1]
```diff
@@ -4444,6 +4444,32 @@
...
+            "slogan": "NLP for Middle Earth",
+            "description": "Hobbit spaCy is a custom spaCy pipeline designed specifically for working with Middle Earth and texts from the world of J.R.R. Tolkien.",
+            "github": "wjbmattingly/hobbit-spacy",
+            "pip": "en-hobbit",
+            "code_example": [
+                "import spacy",
+                "from spacy import displacy",
+                "",
+                "nlp = spacy.load('en_hobbit')",
+                "doc = nlp(text)"
```
Comment (rmitsch) [inline website/meta/universe.json:4460]: The code example is supposed to run as-is. Do you want to define `text` here to demonstrate that the pipeline works?

**Review by rmitsch — COMMENTED** [review]
Thanks for the PR, `hobbit-spacy` looks like a cool project! I'm sure JRRT himself would enjoy it :wink: Just a minor comment.

File: website/meta/universe.json (hunk 1 above)
Comment (rmitsch) [reply website/meta/universe.json:4460]: And the `displacy` import could be removed as well, since you aren't visualizing anything here.

Comment (wjbmattingly) [reply website/meta/universe.json]: Thanks! I have made those two corrections.

**Review by wjbmattingly — COMMENTED** [review]
Hobbit-spacy code corrected

**Review by rmitsch — APPROVED** [review]

**Pull request merged**